src/tests/functional/plugin/shared/include/behavior/ov_executable_network/exec_network_base.hpp

```diff
@@ -80,6 +77,12 @@ TEST_P(OVExecutableNetworkBaseTest, canLoadCorrectNetworkToGetExecutable) {
     EXPECT_NO_THROW(auto execNet = core->compile_model(function, targetDevice, configuration));
 }
 
+TEST(OVExecutableNetworkBaseTest, smoke_LoadNetworkToDefaultDeviceNoThrow) {
+    std::shared_ptr<ov::runtime::Core> core = utils::PluginCache::get().core();
+    std::shared_ptr<ov::Model> function = ngraph::builder::subgraph::makeConvPoolRelu();
+    EXPECT_NO_THROW(auto execNet = core->compile_model(function));
```

Comment:
You do not use `targetDevice` in this test. As a default device will be used `CPU`. It means what these tests should be moved to CPU specific tests. It's not a part of the shared lib (e.g. CPU tests should not be ran on Myriad)

Reply:
Default device will be AUTO when user do not use targetDevice.